akka-actor/src/main/scala/akka/dispatch/affinity/AffinityPool.scala

```diff
@@ -42,13 +39,6 @@ private[affinity] object AffinityPool {
   // PoolState: all threads have been stopped, does not process tasks and does not accept new ones
   final val Terminated = 5
 
```

Comment:
i remember this line well from time with native image :)

Reply:
#PTSD